### ISSUE TITLE
Attachments randomly added to replies [v1.10-rc2]

### ISSUE THREAD

**Issue by kayla-johnson-jchem** [osTicket/osTicket#3112]
Hello,

One thing I have noticed and have yet to see on the issues form is about this attachment issue. I have noticed that one specific attachment will randomly add itself to replies and can't be seen until AFTER a reply is sent. I understand that v1.10 is still in the developmental stages and not recommended for production usage so we are okay with that. Is there a fix for this already that anyone is aware of? 

Thank you,
Kayla

**Comment by ntozier** [comment]
whats the attachment?
Are you sure that the attachment isn't being added by your mail server?

**Comment by kayla-johnson-jchem** [comment]
A few months ago, I added an attachment (meeting notes) to a ticket to keep
track of. The attachment was added the normal way and saved on the osTicket
database. For some reason, it is the only attachment that will randomly
attach itself to tickets. No other attachment added the same way will add
itself.

Kayla Johnson
Instructional Technologist I
IT Specialist
JC:HEM
Skype: kayla.johnson_jchem
Phone: 1-304-905-9361
